src/test/java/chess/domain/piece/PieceTest.java

```diff
@@ -0,0 +1,47 @@
+package chess.domain.piece;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PieceTest {
+
+    Piece piece1;
```

Comment:
```suggestion
    Piece blackQueen;
```

piece1은 아무런 정보도 담지 않는 의미가 없는 네이밍으로 보여지네요 T.T
네이밍을 통해 의도를 들어내면 좋겠어요!